src/satosa/micro_services/processors/scope_remover_processor.py

```diff
@@ -0,0 +1,15 @@
+from ..attribute_processor import AttributeProcessorError
+from .base_processor import BaseProcessor
+
+class ScopeRemoverProcessor(BaseProcessor):
+    """
+    Removes the scope from a given attribute 
+    
+    """
+    def process(self, internal_data, attribute, **kwargs):
+        attributes = internal_data.attributes
+        new_values = []
+        for value in attributes.get(attribute, [None]):
+            if '@' in value:
+                new_values.append(value.split('@')[0])
+        attributes[attribute] = new_values
```

Comment:
The idea is that the deployer uses this microservice for attributes that are scoped. I agree though that removing the attribute entirely in case it is not scoped, doesn't make much sense and will just add to confusion. I will revisit this